test/unit/lib/plugins/package/lib/packageService.test.js

```diff
@@ -523,64 +155,73 @@ describe('lib/plugins/package/lib/packageService.test.js', () => {
     });
   });
 
-  describe.skip('TODO: individually', () => {
+  describe('individually', () => {
+    let fnIndividualZippedFiles;
+    let serverless;
+
     before(async () => {
-      await runServerless({
+      const {
+        fixtureData: { servicePath },
+        serverless: serverlessInstance,
+      } = await runServerless({
         fixture: 'packaging',
         cliArgs: ['package'],
+        awsRequestStubMap: mockedDescribeStacksResponse,
         configExt: {
           package: {
             individually: true,
-            artifact: 'artifact.zip',
             exclude: ['dir1/**', '!dir1/subdir3/**'],
             include: ['dir1/subdir2/**', '!dir1/subdir2/subsubdir1'],
           },
           functions: {
             fnIndividual: {
               package: { include: 'dir1/subdir4/**', exclude: 'dir3/**' },
             },
+            fnService: {
```

Comment:
As discussed, I think we can remove it from `configExt` as it's defined in `packaging` fixture so it should already be there

Reply:
Let me confirm that.

Reply:
Yep, it works correctly.

Reply:
Perfect, thank you so much for checking that :bow: